server/src/main/java/org/opensearch/action/support/IndicesOptions.java

```diff
@@ -167,6 +167,10 @@ public enum Option {
         EnumSet.of(Option.ALLOW_NO_INDICES, Option.IGNORE_UNAVAILABLE),
         EnumSet.of(WildcardStates.OPEN, WildcardStates.CLOSED, WildcardStates.HIDDEN)
     );
+    public static final IndicesOptions LENIENT_EXPAND_OPEN_FORBID_CLOSED = new IndicesOptions(
```

Comment:
did we check the behavior in existing _cat/shards API for hidden or system indices? dothey show up by default?

Reply:
@shwetathareja yes, they do get displayed as part of `_cat/shards` by default.

Reply:
- Should we remove `Option.FORBID_CLOSED_INDICES` from the options  and have only `EnumSet.of(Option.ALLOW_NO_INDICES)` for first argument.
- Should we add `WildcardStates.CLOSED` to expand the closed index also ?

> yes, they do get displayed as part of _cat/shards by default

How are the hidden/system index getting displayed without specifying the option `WildcardStates.HIDDEN`

Reply:
> Should we remove Option.FORBID_CLOSED_INDICES from the options and have only EnumSet.of(Option.ALLOW_NO_INDICES) for first argument.

This is done to NOT fetch and simply ignore the closed indices.

> Should we add WildcardStates.CLOSED to expand the closed index also ?

Kept this to keep it consistent with the existing behaviour of _cat/shards. I'm thinking to keep it as is for now, and re-visit later once we decide on a unified behaviour for both the APIs. But open to suggestion, please let me know if we should add this now itself.

> How are the hidden/system index getting displayed without specifying the option WildcardStates.HIDDEN

They just get shown in the response without any stats similar to closed shards.